core/dbt/config/runtime.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, project_name, version, project_root, source_paths,
         self.validate()
 
     @classmethod
-    def from_parts(cls, project, profile, args):
+    def from_parts(cls, project, profile, args, allow_archive_blocks=False):
```

Comment:
This should probably be something like `allow_archive_configs`, since we're using "blocks" to refer to Jinja tags... whew!

Not worth changing this since it's strictly internal and will be removed in a future release, but let's reserve "blocks" to refer to jinja tags henceforth!

Reply:
I'll actually just change this while I'm fixing your other comment.